model/src/commonMain/kotlin/CountByCategory.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.model
+
+/** Wrapper class for a map of counts by category. */
+data class CountByCategory<T>(
+    /** The map of counts by category. */
+    val map: Map<T, Long>
+) {
+    /** Get the count for the given [key]. */
+    fun getCount(key: T): Long = map[key] ?: 0
```

Comment:
This is completely fine, just to share an alternative Kotlin syntax which might be more speaking: `getOrDefault(key, 0)`